chapter2/tests/t3/chapter2-t3-velero-eks-diff.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# list of kubernetes namespaces that are not backed up
+
+# download velero backup
+VELERO_BACKUP_NAMESPACES=$(curl -sSL https://gist.github.com/dmitry-mightydevops/016139747b6cefdc94160607f95ede74/raw/velero.yaml)
+
+# check for file velero.yaml existence
+if [[ "$VELERO_BACKUP_NAMESPACES" == "404: Not Found"  ]]; then
+  echo "ERROR: file velero.yaml not loaded"
+  exit 1
+fi
+
+# list of kubernetes namespaces backed up
+VELERO_MANIFEST=($(echo "$VELERO_BACKUP_NAMESPACES" | yq ".spec.source.helm.values" \
```

Comment:
This var contains not velero manifest, but namespaces from velero manifest, so suggest to call it like this 
```suggestion
VELERO_NAMESPACES=($(echo "$VELERO_BACKUP_NAMESPACES" | yq ".spec.source.helm.values" \
```

Reply:
done